### PR TITLE
Fix spelling and extend README

### PR DESCRIPTION
**Pull request by zaucker**

### CODE REVIEW

File: README.pod [hunk 1]
```diff
@@ -37,17 +37,29 @@ Curently this includes:
...
 
+Upon startup a config file GpfMeterConfig.json is created in the
+directory where the GpfMeter executable resides. After modification of
+the config file GpfMeter must be restarted for the changes to become
+effective.
+
+=head1 USAGE
+
+The program is probably best started automatically upon login. An icon
+is placed onto the Windows task bar. When the icon is clicked a window
```
Comment (oetiker) [inline README.pod]: the icon is placed into the notification area ...

File: README.pod [hunk 2]
```diff
@@ -37,17 +37,29 @@ Curently this includes:
...
+improving the user experience.
 
 =head1 INSTALLATION
 
-GpfMeter is targetted at windows users. For easy deployment it is written in
+GpfMeter is targeted at Windows users. For easy deployment it is written in
 C#.  You should be able to import this repository into VisualStudio and
 compile your own GpfMeter.
 
+Upon startup a config file GpfMeterConfig.json is created in the
```
Comment (oetiker) [inline README.pod]: configuration file called F<GpfMeterConfig.json> ...

**Comment by zaucker** [comment]
Updated the README ...

**Comment by oetiker** [comment]
thanks!

**Pull request merged**